RecoLocalCalo/HcalRecProducers/python/hfQIE10Reco_cfi.py

```diff
@@ -1,15 +1,14 @@
 import FWCore.ParameterSet.Config as cms
+import RecoLocalCalo.HcalRecProducers.hosimplereco_cfi as _mod
 
-hfQIE10Reco = cms.EDProducer("HcalSimpleReconstructor",
-    correctionPhaseNS = cms.double(0.0),
-    digiLabel = cms.InputTag("simHcalUnsuppressedDigis","HFQIE10DigiCollection"),
-    Subdetector = cms.string('HFQIE10'),
-    correctForPhaseContainment = cms.bool(False),
-    correctForTimeslew = cms.bool(False),
-    dropZSmarkedPassed = cms.bool(True),
-    firstSample = cms.int32(2),
-    samplesToAdd = cms.int32(1),
-    tsFromDB = cms.bool(True)
+hfQIE10Reco = _mod.hosimplereco.clone(
```

Comment:
Please clone from `hfsimplereco` [1] instead: this is `hf`, in fact.
Moreover, there will be a lower number of parameters to be modified.

[1] https://cmssdt.cern.ch/lxr/source/cfipython/RecoLocalCalo/HcalRecProducers/hfsimplereco_cfi.py